include/meshio/meshio_defines.hpp

```diff
@@ -7,8 +7,8 @@
  * file.
  */
 
-#ifndef __MESHIO_DEFINES_HPP__
-#define __MESHIO_DEFINES_HPP__
+#ifndef MESHIO_MESHIO_DEFINES_HPP_
+#define MESHIO_MESHIO_DEFINES_HPP_
```

Comment:
Following [google C++ style guide ](https://google.github.io/styleguide/cppguide.html), MESHIO is the project name and MESHIO_DEFINES_HPP is the header file...